### PR TITLE
Add overlay editor to hypo injectors

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
Makes it possible to change those 2 letters on hypos
## Why It's Good For The Game
You know what you inject without examining the hypo
## Changelog
:cl:
add: hypo overlay Overlay [not a joke]

/:cl:

### CODE REVIEW

**Comment by BlueHNT** [comment]
Why the fuck are these commits here?

**Comment by BlueHNT** [comment]
Reverted the accidental playground merge back to original.. Hooray

**Comment by LeAntek** [comment]
based pr speedmerg

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -45,6 +45,14 @@
 	name = "[core_name] ([str])"
 	label = str
 
+/obj/item/reagent_containers/hypospray/proc/overlay(mob/user)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:48]: why is this a seperate proc just inline it

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -45,6 +45,14 @@
...
+	var/str = copytext(reject_bad_text(input(user,"Hypospray tag text?", "Set tag", "")), 1, MAX_NAME_HYPO)
+	if(!length(str))
+		to_chat(user, span_notice("Invalid text."))
+		return
+	to_chat(user, span_notice("You tag [src] as \"[str]\"."))
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:53]: balloon alert?

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Rest ok

**Pull request merged**